package.json

```diff
@@ -18,11 +20,13 @@
     "lighthouse": "node ./jobs/lighthouse.js",
     "dev": "next dev",
     "build": "next build",
-    "start": "next start -p $PORT"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
+    "start": "next start -p $PORT",
+    "lint": "npx eslint ./ --fix",
+    "test:lint:fix": "npx eslint ./ --fix",
+    "test:lint:css": "stylelint  \"**/*.css\" --fix",
```

Comment:
rename test:lint:fix to lint:fix . do the same with css and rename test command to lint:all

Reply:
done